source/fundamentals/encrypt-fields.txt

```diff
@@ -45,10 +45,10 @@ uniquely.
 
 The MongoDB manual contains detailed information on the following {+qe+} topics:
 
-- Begin a working {+qe+} example with the :v6.0:`queryable-encryption quick start </core/queryable-encryption/quick-start>`.
-- Learn how to use {+qe+} in the :v6.0:`queryable-encryption fundamentals </core/queryable-encryption/fundamentals>`.
-- Learn how to integrate your {+qe+} implementation with a KMS in the :v6.0:`queryable-encryption tutorials </core/queryable-encryption/tutorials>`.
-- Learn {+qe+} concepts in the :v6.0:`queryable-encryption reference </core/queryable-encryption/reference>`.
+- To get started, see the {+qe+} :v6.0:`Quick Start </core/queryable-encryption/quick-start>`.
+- To learn how to use {+qe+}, see the {+qe+} :v6.0:`Fundamentals </core/queryable-encryption/fundamentals>`.
+- To learn how to integrate your {+qe+} implementation with a KMS, see the {+qe+} :v6.0:`Tutorials </core/queryable-encryption/tutorials>`.
+- To learn {+qe+} concepts, see the {+qe+} :v6.0:`Reference </core/queryable-encryption/reference>`.
```

Comment:
Issue: linked text should contain the identifying info (product name in this case)

Suggestion:
```
:v6.0:`{+qe+} Reference</core/queryable-encryption/reference>`.
```